docs/.vuepress/guide.js

```diff
@@ -19,6 +19,10 @@ sidebar = [
         text: 'Laravel',
         link: '/guide/laravel.md',
     },
+    {
+        text: 'Generate CRUD',
+        link: '/guide/generate_crud.md',
```

Comment:
@wri-abinash-s001 @we-vikram-wri231 instead of `_` use '-` in url slugs

Reply:
@themodernpk Ok Sir
we will update this

Reply:
@themodernpk noted sir